core/gui/src/app/workspace/component/workflow-editor/context-menu/context-menu/context-menu.component.html

```diff
@@ -109,7 +109,7 @@
 
   <li
     nz-menu-item
-    *ngIf="operatorMenuService.highlightedOperators.value.length === 1 && isWorkflowModifiable"
+    *ngIf="isSelectedOperatorValid()"
```

Comment:
Can we disable the button instead of hiding it? We can align the design with the run button, i.e., the run button becomes disabled when the validation fails.

Reply:
Resolved!